WorkflowAdminBundle/DependencyInjection/OpenOrchestraWorkflowAdminExtension.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace OpenOrchestra\WorkflowAdminBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\DependencyInjection\Loader;
+
+/**
+ * This is the class that loads and manages your bundle configuration
+ *
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
+ */
+class OpenOrchestraWorkflowAdminExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('form.yml');
+        $loader->load('leftpanel.yml');
```

Comment:
I'm totally for making a general refacto, I'll add that ticket.

My point was to stop using this wrong terminology when we're creating new files and can avoid it, in order to simplify this future refacto and initiate the move :)